services/brig/src/Brig/App.hs

```diff
@@ -576,6 +583,16 @@ newtype HttpClientIO a = HttpClientIO
       MonadIndexIO
     )
 
+type BrigCanonicalEffects =
+  '[ PasswordResetStore,
+     Now,
+     CodeStore,
+     Embed Cas.Client, -- FUTUREWORK: remove this effect once MonadClient
```

Comment:
I'm not sure this FUTUREWORK is correct; I think you'll want to keep this forever.

Reply:
You're right: code store is interpreted to `Embed m` for `MonadClient m`, e.g., `Cas.Client`.

Fixed in e090dd9b8c8ade4736a4e21111861ba6204d55d8.